sections/discussion/category-topic.tex

```diff
@@ -0,0 +1,46 @@
+\subsection{Category-topic model}\label{sec:discussion_category_topic} 
+Specific observations for the category-topic model can also be made.
+Just as for the author-topic model, the similarity between pairs of categories can be calculated.
+Because topic distributions are generated for each category, just as they are for each author in the author-topic model, \autoref{eq:author_similarity} can be used for categories as well, with $i$ and $j$ being the categories.
+In \autoref{tab:category_similarity}, the top 10 category pairs, based on symmetric KL divergence, are shown.
+
+\begin{table}[h]
+	\centering
+	\caption{Top 10 category pairs based on the symmetric KL divergence between categories.}
+	\begin{tabular}{r|c}
+		Category pair & KL \\
+		\midrule
+		misc (292) \& Friii (2333) & 3.65 \\
+		Friii (2333) \& Debat (10075) & 3.80 \\
+		Feature (188) \& Hjørring-avis (4235) & 4.22 \\
+		Sport-avis (10941) \& Morsø Sport (2350) & 5.04 \\
+		Indsigt (984) \& Perspektiv (613) & 5.27 \\
+		53. Frederik (203) \& Navne (3749) & 5.69 \\
+		Rebild-avis (4415) \& Bo Godt (1447) & 5.78 \\
+		Nordjyske Biler (1400) \& Thisted-avis (11473) & 5.91 \\
+		misc (292) \& Debat (10075) & 6.67 \\
+		Frederikshavn-avis (4325) \& Bo Godt (1447) & 6.81 \\
+		\midrule
+		Maximum KL divergence & 35.62 \\
+		Median KL divergence & 27.09 \\
+	\end{tabular}
+	\label{tab:category_similarity}
+\end{table}
+
+The second most similar pair, 'Friii' and 'Debat', is interesting to look at since 'Debat' has a specific writing style and 'Friii' does not.
+While 'Friii' does not seem to have a theme in the articles written, articles with the 'Debat' (debate) category seem to mostly cover themes that can bring differing opinions and articles with interviews.
```

Comment:
```suggestion
The second most similar pair, 'Friii' and 'Debat', is interesting to look at since 'Friii' does not seem to have a theme in the articles written, articles with the 'Debat' (debate) category seem to mostly cover themes that can bring differing opinions and articles with interviews.
```